apps/pie-docs/src/assets/js/accordions.js

```diff
@@ -122,27 +128,11 @@ class Accordion {
 }
 
 function initialise () {
-    const accordions = [];
-
-    const resizeObserver = new ResizeObserver((entries) => {
-        entries.forEach((entry) => {
-            const { width } = entry.contentRect;
-
-            // ensure we don't duplicate accordion instances
-            if (width > 600 && !accordions.length) {
-            // TODO: use fozzie JS breakpoint helpers instead of hardcoding 600px
-                document.querySelectorAll('.c-nav details').forEach((el) => {
-                    accordions.push(new Accordion(el));
-                });
-            }
-        });
-    });
```

Comment:
we don't need this logic anymore?

Reply:
Thanks @raoufswe I left a comment around this in the description :) , here it is:

There is no record of the decision on why we implemented the animation only for desktop and not mobile screens. As this solution relies on the initial animation script, I removed the check for smaller screens (>=600). Without this check I could not see a use for the resizeObserver either. I wasn't able to understand why we would need it before as we set the accordions once and do not change them on resize any more. If anybody has more context let me know.

Do you know why we need it or remember context around this decision?

Reply:
maybe @jamieomaguire has any idea? 👀

Reply:
Ash asked for it, I delivered :P 

In all seriousness, it was something we felt was a nice UX improvement. Why we left out mobile, I cannot remember unfortunately. 

I think it is perfectly fine for us to remove this for now and revisit after chatting with design :)